### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'package_name'

### ISSUE THREAD

**Issue by jiep** [moritzgloeckl/overleaf-sync#31]
After updating to new version `1.1.5` and running `ols` the following error is raised:

```
$ ols
Traceback (most recent call last):
  File "/usr/local/bin/ols", line 5, in <module>
    from olsync.olsync import main
  File "/usr/local/lib/python3.8/dist-packages/olsync/olsync.py", line 48, in <module>
    def main(ctx, local, remote, project_name, cookie_path, sync_path, olignore_path, verbose):
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 282, in decorator
    return option(*(param_decls or ('--version',)), **attrs)(f)
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 173, in decorator
    _param_memo(f, OptionClass(param_decls, **option_attrs))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1547, in __init__
    Parameter.__init__(self, param_decls, type=type, **attrs)
TypeError: __init__() got an unexpected keyword argument 'package_name'
```

Python version: `3.8.6`
OS: `Ubuntu 20.04.1 LTS`

**Comment by rayvburn** [comment]
Although I didn't use any older version, I can confirm that I got the same error message with 1.1.5. The same error shows up after calls:

```bash
ols login
```

and

```bash
ols
```

Python version: `3.8.10`
OS: `Ubuntu 20.04.2 LTS`

**Comment by jiep** [comment]
> I've checked closed issues, @jiep, see #30, in particular this [#30 (comment)](https://github.com/moritzgloeckl/overleaf-sync/issues/30#issuecomment-878729144). This seems to fix the issue. In my case it was line number 46.

Thank you, @rayvburn! It works!
